app/views/categories/index.html.erb

```diff
@@ -21,7 +21,7 @@
       <% @category.each do |category| %>
         <tr>
           <td><%= category.id %></td>
-          <td><%= link_to category.title %></td>
+          <td><%= link_to category.title, admins_path %></td>
```

Comment:
use this code instead:
`<td><%= link_to category.title, words_path(category) %></td>`